exercises/object-oriented-programming/src/main/java/com/cbfacademy/dogs/Main.java

```diff
@@ -0,0 +1,21 @@
+package com.cbfacademy.dogs;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        Dog dogOne = new Dog("poodle", 'L', 4, "black");
+        Dog dogTwo = new Dog("lab", 'M', 5, "white"); 
+        Dog dogThree = new Dog ("bulldog", 'M', 2, "brown");
+
+
+        dogOne.run();
```

Comment:
You're being warned about this line because you've defined a class-level (static) method, but you're calling it in a non-static way, meaning against an instance of the class, rather than on the class itself.

A static call would look like 

```java
Dog.run();
```

So remembering that static members are shared across all instances of a class, should `run()` and the other methods be static?

Reply:
Ok thank you for the clarification!